boards/arm/v2m_musca/v2m_musca.dts

```diff
@@ -0,0 +1,32 @@
+/dts-v1/;
+
+#include <arm/armv8-m.dtsi>
+
+#include "v2m_musca-secure.dtsi"
```

Comment:
we can steer the secure/non-secure here by ifdef'ing this with CONFIG_ARM_NONSECURE_FIRMWARE to select the non-secure one.

Reply:
Agree.